src/fixup.js

```diff
@@ -297,6 +297,40 @@
     request.send();
   }
 
+  /* Dark mode toggle */
+  const darkCss = document.querySelector('link[href="dark.css"]');
+  if (darkCss && matchMedia("(prefers-color-scheme)").matches) {
+    const colorScheme = localStorage.getItem("tr-theme") || "auto";
+    darkCss.disabled = colorScheme === "light";
+    darkCss.media = colorScheme === "auto" ? "(prefers-color-scheme: dark)" : "";
+    const render = document.createElement("x-temp");
```

Comment:
why `x-temp` rather than `div`?

Reply:
Fsmhjh

Reply:
No reason... I just like using generic custom element, but a `div` would work just as well.

Reply:
@marcoscaceres I feel like div is better than "generic custom element"... If you're going to make a custom element, at least have a decent justification for it!

Reply:
addressed: https://github.com/w3c/tr-design/pull/243/commits/9efe263c396ab35f998f9d313532176e6851367e